A Cassandra CQL returner

ckochenower opened the pull request:
This returner cross-calls modules/cassandra_cql.py

ckochenower committed:
A Cassandra CQL returner

This returner cross-calls modules/cassandra_cql.py

ckochenower committed:
A Cassandra CQL returner

This returner cross-calls modules/cassandra_cql.py

cachedout commented on salt/returners/cassandra_cql_return.py:
```diff
@@ -0,0 +1,390 @@
+    try:
+        data = __salt__['cassandra_cql.cql_query'](query)
+        if data:
+            load = data[0].get('load')
```
This might need more error checking. Could see other exceptions floated from module and ```data``` could be a zero-item list.

cachedout commented:
Very nice.

Commit:
Merge remote-tracking branch 'upstream/develop' into develop

Commit:
Make nocache parameter an optional parameter

The Salt SSH client calls prep_jid without any parameters.
As of this commit, the following python modules define
prep_jid with an optional nocache parameter.

returners/
    couchbase_return.py
    local_cache.py
    multi_returner.py
    postgres_local_cache.py

A of the remaining returners require the nocache parameter
and therefore are incompatible with the Salt SSH client.

To allow the Salt SSH client to work with all returners,
make the nocache parameter in all returners optional.

Commit:
Make nocache parameter an optional parameter

The Salt SSH client calls prep_jid without any parameters.
As of this commit, the following python modules define
prep_jid with an optional nocache parameter.

returners/
    couchbase_return.py
    local_cache.py
    multi_returner.py
    postgres_local_cache.py

A of the remaining returners require the nocache parameter
and therefore are incompatible with the Salt SSH client.

To allow the Salt SSH client to work with all returners,
make the nocache parameter in all returners optional.

cro commented on salt/utils/templates.py:
```diff
@@ -449,7 +449,7 @@ def py(sfn, string=False, **kwargs):  # pylint: disable=C0103
 
     try:
-        data = mod.run()
+        data = mod.run()str0ng
```
Typo @ckochenower ?

ckochenower replied:
Yep. Can you tell I didn't test this commit? Reminder to self. No matter how trivial a change. Test it!

ckochenower committed:
Merge branch 'develop' of https://github.com/saltstack/salt into develop

ckochenower committed:
Remove eroneous text causing test failures

ckochenower committed:
Remove eroneous text causing test failures

ckochenower committed:
Merge branch 'develop' of https://github.com/ckochenower/salt into develop

ckochenower committed:
Fix lint errors

ckochenower committed:
The last of the lint

thatch45 commented:
@ckochenower wins!
